src/shared/bridge-events.js

```diff
@@ -11,6 +11,11 @@ export default {
    */
   FEATURE_FLAGS_UPDATED: 'featureFlagsUpdated',
 
+  /**
+   * Focus the annotations indicated by the passed array of $tags
+   */
+  FOCUS_ANNOTATIONS: 'focusAnnotations',
```

Comment:
I don't know the relative value of this layer of abstraction for bridge-event names, but I figure, may as well try to be consistent one way or the other.